### PR TITLE
Admin Panel Module

### PR DESCRIPTION
**Pull request by ronakjain2012**
Start...

### CODE REVIEW

**Comment by hackerfrog** [comment]
@ronakjain2012 please sign the CLA which is attached to this pull request. [CLA Link](https://cla-assistant.io/hackerfrog/TH?pullRequest=7)

**Comment by hackerfrog** [comment]
I had checked your work it's quite impressive, but you need to improve your designing skills.
I am attaching image to make page more styles.
admin_layout

And some points I had mad so you can change into your Admin Module.
* Rename folder name : `Admin Panel` :arrow_right: `admin` 
* Rename file name : `adminp.php` :arrow_right: `index.php` & `LandingPage.php` :arrow_right: `dashboard.php`
* 
* **Features Bars Items**
  * Dashboard
  * Users
     - [ ] Add User
     - [ ] View Users
  * Locations `only for [super admin] & [admin]`
  * Routes
     - [ ] New Route
     - [ ] View Routes
  * Buses
     - [ ] New Bus
     - [ ] View Buses
  * Tickets
     - [ ] Book Ticket
     - [ ] Tickets Details
  * Transactions
     - [ ] Tickets Transactions
     - [ ] Agency Transactions
  * Coupons
     - [ ] New Coupons
     - [ ] Coupons Details
  * Settings

**Comment by ronakjain2012** [comment]
Changes Done ! Check and Update Me

**Comment by hackerfrog** [comment]
@ronakjain2012 
Some changes are left in dashboard.php to change encryption type from `SHA256` to `MD5`

File: AdminPanel/dashboard.php [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+function add_user($Uname,$Upass,$Urole) {
+	try {
+		$host = "localhost";
+		$user = "root";
+		$pwd = "root";
+		$db = "adminp";
+		$con = new MySQLi($host,$user,$pwd);		
+		$con->select_db($db);
+		$Uname1 = $Uname;
+		$Uname = hash('sha256',$Uname);
```
Comment (hackerfrog) [inline AdminPanel/dashboard.php]: please change the encryption to MD5 type..

**Comment by ronakjain2012** [comment]
Done ! MD5 added to DB file as will as in Authentication

**Pull request merged**